client/src/components/App.js

```diff
@@ -26,6 +28,7 @@ const App = (props) => {
         <Route exact path="/">
           <h2>Hello from react</h2>
         </Route>
+        <Route exact path="/index" component={IndexPage}/>
```

Comment:
Might want to name this path `/trips` since that is what is displayed at that route, `index` is a bit obscure